config/locales/en.yml

```diff
@@ -17,6 +17,7 @@ en:
         peer_review_results: "Peer Review Results"
         your_reviews: "Your Reviews"
         review: "Peer Review"
+        deadline_with_extension: "%{deadline} (you have an extension until %{extension_deadline})"
```

Comment:
Please add this under `locales/views/assignments/en.yml` (the key will be `assignments.deadline_with_extension)`.